src/modules/hotkeys/assets/commands/schoolings.ts

```diff
@@ -0,0 +1,12 @@
+import type { Empty, Scope } from 'typings/modules/Hotkeys';
+
+export default <Scope<Empty, [], [], true>>{
+    educate() {
+        // #multiple_commits is the button added by https://forum.leitstellenspiel.de/index.php?thread/23203-mehr-als-4-klassenr%C3%A4ume-zeitgleich-nutzen/
+        document
+            .querySelector<HTMLInputElement | HTMLSpanElement>(
```

Comment:
Why do we check on  a specific script?

Reply:
for compatibility reason. The script removes the default button thus the hotkey wouldn't work anymore.